sky/cli.py

```diff
@@ -2635,6 +2635,53 @@ def storage_delete(names: Tuple[str], all: bool):  # pylint: disable=redefined-b
         sky.storage_delete(name)
 
 
+@storage.command('create', cls=_DocumentedCodeCommand)
+@click.option(--'name',
+              type=str,
+              default=None,
+              flag_value=None,
+              nargs=1)
+@click.option('--source',
+              type=str,
+              default=None,
+              flag_value=None,
+              nargs=1)
+@click.argument('stores',
+                default=['S3'],
+                required=False,
+                type=str,
+                nargs=-1)
+@usage_lib.entrypoint
+def storage_create(name: str, source: str, stores: Tuple[str]):
+    """Create a storage object.
+    Args:
+          name: str; Name of the storage object (must specify option with --name)
+          source: str; File path where the data is initially stored. Can be a
+            local path or a cloud URI (s3://, gs://, etc.). Local paths do not
+            need to be absolut (must specify option with --source)
+          stores: Optional; Specify pre-initialized stores (S3 or GCS).
+
+    Examples:
+
+    .. code-block:: bash
+
+        # Create an empty storage object named imagenet with GCS and S3 stores
+        sky storage create --name imagenet GCS S3
+
+        # Create a storage object from bucket URL using source bucket as the backing store
+        sky storage create --source S3://bucket-name
```

Comment:
Make sure to run these commands before putting them as example^ Name is required and name is not specified here